tests/Utils/Test_SafeMath.cpp

```diff
@@ -792,6 +860,23 @@ BOOST_AUTO_TEST_CASE(test_boost_int128_t) {
                                    num1, num2, mulRes6),
                       "Test mul underflow failed!");
 
+  boost::multiprecision::int128_t subRes;
+  BOOST_CHECK_MESSAGE(
+      true == SafeMath<boost::multiprecision::int128_t>::sub(
+                  0,
+                  std::numeric_limits<boost::multiprecision::int128_t>::min(),
+                  subRes),
+      "Test signed sub overflow failed!");
+  BOOST_REQUIRE(std::numeric_limits<boost::multiprecision::int128_t>::max() ==
+                subRes);
+
+  BOOST_CHECK_MESSAGE(
+      false == SafeMath<boost::multiprecision::int128_t>::sub(
+                   1,
+                   std::numeric_limits<boost::multiprecision::int128_t>::min(),
+                   subRes),
+      "Test signed sub overflow failed!");
+
```

Comment:
May we need to check `-1-min` for `int128_t`, `int256_t`, `int512_t`, and `int1024_t`?

Reply:
I change the test case because for boost int128_t the negative range is same as positive range